test/e2e/runner/start.go

```diff
@@ -66,8 +73,26 @@ func Start(testnet *e2e.Testnet) error {
 		}
 	}
 
-	// Start up remaining nodes
 	for _, node := range nodeQueue {
+		if node.StartAt > networkHeight {
+			// if we're starting a node that's ahead of
+			// the last known height of the network, then
+			// we should make sure that the rest of the
+			// network has reached at least the height
+			// that this node will start at before we
+			// start the node.
```

Comment:
As you noted, I had it completely backward. Thanks for the clarification. 

Given that we're concerned with _any_ node past the target network height, could we save some time here by finding the maximum value among all the queued nodes before waiting? As written, we will wait each time we find some node past the target network height—but I think we only really care about the furthest forward, is that right?

Reply:
I think I like the approach of waiting in smaller chunks (it means we wait a few times for each value and then start a node and then wait a bit longer) rather than risk hitting the timeout while we wait for the node to get to the maximal height. 

In some ways, I think the fact that my change is theoretically a lot slower than the solution you propose is intentional/desirable,

Reply:
I don't think the performance really matters that much given what we're using this for. I mostly thought it might be easier to debug if something goes wrong in _one_ wait cycle than in potentially several of them. But your point about early detection makes sense.